cli/src/main/java/hudson/cli/CLI.java

```diff
@@ -246,7 +245,7 @@ public static int _main(String[] _args) throws Exception {
         if (auth == null && bearer == null) {
             // -auth option not set
             if ((userIdEnv != null && !userIdEnv.isBlank()) && (tokenEnv != null && !tokenEnv.isBlank())) {
-                auth = StringUtils.defaultString(userIdEnv).concat(":").concat(StringUtils.defaultString(tokenEnv));
+                auth = userIdEnv.concat(":").concat(tokenEnv);
```

Comment:
I think you should use `Objects.toString(s, "");` for `StringUtils.defaultString(s);` as stated [here](https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/StringUtils.html#defaultString-java.lang.String-java.lang.String-) which is also deprecated in commons lang 3

```suggestion
                auth = Objects.toString(userIdEnv, "").concat(":").concat(Objects.toString(tokenEnv, ""));
```

Reply:
I modified it in two other places.

However, Considering the context here, it is located within the `if` block of `!= null`. So there's no need to wrap it with `defaultString`.